docker/Dockerfile.jest

```diff
@@ -0,0 +1,7 @@
+FROM node:10.15-alpine
```

Comment:
Do we need this dockerfile? What's the difference between this and [Dockerfile.webpack](https://github.com/metabrainz/listenbrainz-server/blob/52705348/docker/Dockerfile.webpack#L1) and could we just use the same dockerfile for both running tests and compiling the jsx?